rest/taskana-rest-spring/src/test/java/pro/taskana/user/rest/UserControllerRestDocTest.java

```diff
@@ -0,0 +1,62 @@
+package pro.taskana.user.rest;
+
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.delete;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.put;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import pro.taskana.common.rest.RestEndpoints;
+import pro.taskana.common.test.BaseRestDocTest;
+import pro.taskana.user.api.UserService;
+import pro.taskana.user.api.models.User;
+import pro.taskana.user.rest.assembler.UserRepresentationModelAssembler;
+import pro.taskana.user.rest.models.UserRepresentationModel;
+
+class UserControllerRestDocTest extends BaseRestDocTest {
```

Comment:
Did you check the created rest-api.doc file? Was it all as expected?

Reply:
I checked it and everything looks good :)

Reply:
perfect